library/alloc/src/boxed.rs

```diff
@@ -358,7 +356,6 @@ impl<T> Box<T> {
     /// # Ok::<(), std::alloc::AllocError>(())
     /// ```
     #[unstable(feature = "allocator_api", issue = "32838")]
-    // #[unstable(feature = "new_uninit", issue = "63291")]
```

Comment:
Hm, are these just stale comments? I'm a bit confused why we're deleting these in *this* PR...

Reply:
These comments refer to the tracking feature that `new_zeroed_alloc` was split out of. The comments on the dual-featured zeroed functions (e.g., `try_new_zeroed`) were not updated when that feature was split, so it makes sense to remove those. Then, if I'm doing that, it seems reasonable to fix it for the rest of the feature.

Reply:
I see. Thanks for clarifying!